src/modules/commander/commander_params.c

```diff
@@ -798,22 +798,22 @@ PARAM_DEFINE_INT32(NAV_DLL_ACT, 0);
 PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
 
 /**
- * Flag to enable obstacle avoidance
+ * Flag to enable obstacle avoidance.
  *
  * @boolean
  * @group Mission
  */
 PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
 
 /**
- * Set onboard controller bootup timeout
+ * Set avoidance system bootup timeout.
  *
- * This parameter defines the bootup timeout.
- * After the timeout, a mavlink message that tells the user that the avoidance system
- * is not available is sent.
+ * The avoidance system running on the companion computer is expected to boot
+ * within this time and start providing trajectory points.
```

Comment:
Is this true? Is it about trajectory points being sent? Or is it about a heartbeat?

Reply:
Yes, it is true. In the actual implementation, as soon as the avoidance node is running sends both heartbeats and trajectory points. On the long term, this behavior will probably be changed.

If we want to maintain a more general statement we could go for: 
`The avoidance system running on the companion computer is expected to boot within this time, after which its functionalities are fully operative.`

Reply:
No if it's true it's fine.